### PR TITLE
Discord: Add column for linking channel to circle

### PR DESCRIPTION
**Pull request by topocount**

### CODE REVIEW

**Comment by topocount** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1830** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/1830" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/coordinape/coordinape/1830?utm_source=stack-comment).

File: api-lib/gql/__generated__/zeus/const.ts [hunk 1]
```diff
@@ -1662,6 +1662,7 @@ export const AllTypesProps: Record<string, any> = {
     circle_id: 'bigint_comparison_exp',
     created_at: 'timestamptz_comparison_exp',
     id: 'bigint_comparison_exp',
+    server_channel: 'String_comparison_exp',
```
Comment (CryptoGraffe) [inline api-lib/gql/__generated__/zeus/const.ts:1665]: Can we call this discord_server_channel instead? otherwise it is pretty ambiguous

**Review by CryptoGraffe — CHANGES_REQUESTED** [review]
I strongly suggest we rename the column to have discord in the name

File: api-lib/gql/__generated__/zeus/const.ts (hunk 1 above)
Comment (karelianpie) [reply api-lib/gql/__generated__/zeus/const.ts:1665]: I'd suggest `discord_channel_id` and `discord_role_id` as that's the nomenclature they use in discord

**Review by CryptoGraffe — APPROVED** [review]
thanks for renaming column. g2g

**Review by karelianpie — APPROVED** [review]
LGTM

**Pull request merged**